modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -0,0 +1,244 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-controller-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-autoscaling-ingress-controller_{context}"]
+= Autoscaling an Ingress Controller
+
+Automatically scale an Ingress Controller to dynamically meet routing performance or availability requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
+
+.Prerequisites
+. You have the OpenShift CLI (`oc`) installed.
+. You have access to an {product-title} cluster as a user with the `cluster-admin` role.
+. You have the Custom Metrics Autoscaler Operator installed.
+
+.Procedure
+. Create a project in the `openshift-ingress-operator` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc project openshift-ingress-operator
+----
+
+. Enable OpenShift monitoring for user-defined projects by creating and applying a config map.
+
+.. Create a new `ConfigMap` object, `cluster-monitoring-config.yaml`:
++
+.cluster-monitoring-config.yaml
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true <1>
+----
++
+<1> When set to `true`, the `enableUserWorkload` parameter enables monitoring for user-defined projects in a cluster.
+
+.. Apply the config map by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f cluster-monitoring-config.yaml
+----
+
+. Create a Service Account to authenticate with Thanos by running the following command:
++
+[source,terminal]
+----
+$ oc create serviceaccount thanos && oc describe serviceaccount thanos
+----
++
+.Example output
+[source,terminal]
+----
+Name:                thanos
+Namespace:           openshift-ingress-operator
+Labels:              <none>
+Annotations:         <none>
+Image pull secrets:  thanos-dockercfg-b4l9s
+Mountable secrets:   thanos-dockercfg-b4l9s
+Tokens:              thanos-token-c422q
+Events:              <none>
+----
+
+. Define a `TriggerAuthentication` object within the `openshift-ingress-operator` namespace using the Service Account's token.
+
+.. Define the variable `$secret` that contains the secret by running the following command:
```

Comment:
I think technically the variable name is just `secret`, and the `$` is used later to access the stored value's variable. So it might be more correct to just call it (for example) "the `secret` variable" within sentences.

Reply:
Good catch! Thanks